app/css/animator.css

```diff
@@ -53,6 +53,47 @@ a {
   text-align: center;
 }
 
+/* ========== CONFIRM BOXES ========== */
+
+#confirm-container {
+    position: absolute;
+    height: 100vh;
+    width: 100vw;
+    background-color: rgba(23, 23, 23, 0.5);
+    z-index: 4;
+    text-align: center;
+}
+
+#confirm-container.hidden { display: none; }
+
+#confirm {
+    position: relative;
+    top: calc(50vh - 10em);
+    display: inline-block;
+    width: 20em;
+    background: #000;
```

Comment:
Should be `background-color`. And to answer your question I never answered, yes, shorthand `background` is used when applied a `background-image` and many of the other `background-*` properties, not when just setting a color.